packages/react/test/utils/ecosystem.ts

```diff
@@ -1,5 +1,5 @@
 import { act } from '@testing-library/react'
-import { createEcosystem } from '@zedux/react'
+import { createEcosystem } from '@zedux/atoms'
```

Comment:
is this new?

Reply:
No I split this out right before Zedux v1. `@zedux/react` has a direct dependency on it, so you never have to know about `@zedux/atoms` if you're using React like us.